docs/molecules/Historypage.md

```diff
@@ -0,0 +1,155 @@
+---
+title: History Page
+sidebar_label: History Page
+---
+
+<head>
+  <title> History Page </title>
+  <meta name="description" content="your meta content goes here" />
+</head>
+
+The ``HistoryPage`` component is a React page that displays a list of historical chat interactions. It allows users to view past conversations and delete them if needed.
+
+<img src= "/img/molecules/historypage.png" alt="History Page Molecule" />
+
+## Usage
+
+```
+import HistoryPage from './historyPage';
+
+const App = () => {
+  return (
+    <HistoryPage />
+  );
+}
+
+export default App;
+```
+
+## Description
+
+The `HistoryPage` molecule consists of the following elements:
+
+- It provides a list of past chat interactions for users to review.
+- Users can delete individual conversations from the history.
+
+## Functionality
+
+ - Users can click on a chat item to view details of the conversation.
+- They can delete conversations by clicking the delete icon and confirming the action.
+
+## Dependencies
+
+- React: JavaScript library for building user interfaces.
+- Material-UI (Mui): React components for faster and easier web development.
+- Lodash: JavaScript utility library for common programming tasks.
+
+ 
+
+## Styling
+
+- CSS modules are utilized for styling the component.
+- Styles are defined in the `style.module.css` file.
+
+## Customization
+
+ The **HistoryPage** component offers flexibility for customization to match your application's requirements. Here are some customization options:
+
+### Theme Customization
+
+Modify the colors and styles in the `style.module.css` file to match your application's design.
+
+
+### Component Configuration 
+
+ Update the component object in the config.json file to customize the title and other configurations
+
+ ```json
+{
```

Comment:
Do not provide admin's config, it is used to render the form, provide bot's config

Reply:
resolved